taxi/forms.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ["license_number", ]

```

Comment:
The `DriverLicenseUpdateForm` is missing validation for the `license_number`. According to the task requirements, it should check for 8 characters, with the first 3 being uppercase letters and the last 5 being digits.